package/crds/compute.ionoscloud.crossplane.io_privatecrossconnects.yaml

```diff
@@ -0,0 +1,353 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.12.1
+  name: privatecrossconnects.compute.ionoscloud.crossplane.io
+spec:
+  group: compute.ionoscloud.crossplane.io
+  names:
+    categories:
+    - crossplane
+    - managed
+    - ionoscloud
+    kind: PrivateCrossConnect
+    listKind: PrivateCrossConnectList
+    plural: privatecrossconnects
```

Comment:
better to have these as pcc and pccs

Reply:
this is auto generated. Where do i have to change those?

Reply:
sadly, I saw that basically change the name of the structures.

Reply:
jup :/

I dont really like acronyms. You have to decide what todo. :)